pkg/util/util.go

```diff
@@ -121,15 +121,15 @@ func GetAvailableSpace(path string) (int64, error) {
 
 // GetAvailableSpaceBlock gets the amount of available space at the block device path specified.
 func GetAvailableSpaceBlock(deviceName string) (int64, error) {
-	// Check if device exists.
-	info, err := os.Stat(deviceName)
+	// Check if the file exists and is a device file.
+	info, err := os.Lstat(deviceName)
```

Comment:
So why change to lstat here? According to the documentation lstat doesn't follow symlinks while stat does. If the file we are looking at is a symlink I think we want to follow it.

Reply:
yes, lstat change is not needed. Reverted.